### PR TITLE
Update defender.yml

### PR DESCRIPTION
**Pull request by Coy47**
Added all of the information for defenders.

### CODE REVIEW

File: src/assets/data/character/defender.yml [hunk 1]
```diff
@@ -66,6 +66,41 @@
     power: ATK x 4500%
     maxHits: 25
 
+  notes: >
+      _A2 arrives excpetionally early for GL along with the other Nier characters. This also makes her the first Greatsword character
```
Comment (seiyria) [inline src/assets/data/character/defender.yml]: Typo, should be exceptionally

File: src/assets/data/character/defender.yml [hunk 2]
```diff
@@ -267,10 +342,6 @@
       effects: 
         - desc: HP +20%
           all: true
-          meta:
```
Comment (seiyria) [inline src/assets/data/character/defender.yml]: Don't remove these. These are important to the party creator.

**Comment by seiyria** [comment]
Please address my comments and also check the deploy preview to make sure all of the notes show up.

File: src/assets/data/character/defender.yml (hunk 2 above)
Comment (Coy47) [reply src/assets/data/character/defender.yml]: I think I started working on this while you were adding party creator so on my version meta wasn't there.

Comment (Coy47) [reply src/assets/data/character/defender.yml]: Also don't approve it defender's aren't appearing on the preview

Comment (seiyria) [reply src/assets/data/character/defender.yml]: Check the indentation of your notes. The notes block should be the same alignment as the rush block. It shouldn't be further in.

**Pull request closed without merging**